shard.yml

```
@@ -1,10 +1,10 @@
 name: slack
-version: 0.9.0
+version: 0.9.1
```

```suggestion
version: 0.10.0
```

Since we are changing the minimum crystal version a non-patch update of the version is needed.